packages/suite/src/utils/wallet/coinjoinUtils.ts

```diff
@@ -336,3 +337,6 @@ export const getIsCoinjoinOutOfSync = (selectedAccount: SelectedAccountStatus) =
         return account.status === 'out-of-sync';
     }
 };
+
+export const getAccountProgressHandle = (account: Account) =>
```

Comment:
nit: I would prefer to pass only what is needed (the account.key)

Reply:
Counter-arguments:
- this way it'd be simpler to change how the progress handle is extracted from the account
- it also serves as sort of type check, because it's harder to accidentally pass wrong account-shaped object to the function than a wrong string

But I have no problem with changing it if you insist :)